.github/workflows/automerge.yml

```
@@ -0,0 +1,43 @@
+name: AutoMerge with 2+ Approvals
+
+on:
+  pull_request_review:
+    types: [submitted]
+jobs:
+  automerge:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check approved reviews count
+        id: check_approvals
+        uses: actions/github-script@v6
```

actionlint (1.7.7)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)